core-java-modules/core-java-streams-6/src/test/java/com/baeldung/streams/inputstreamintostringstream/InputStreamToStringStreamUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package inputstreamintostringstream;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+import java.util.regex.MatchResult;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;

```

Comment:
Using the wrong assertEquals, import the JUnit5 one
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
```